appveyor.yml

```diff
@@ -17,8 +17,8 @@ install:
 - reg ADD "HKLM\Software\Wow6432Node\Microsoft\StrongName\Verification\Microsoft.PackageManagement.NuGetProvider,31bf3856ad364e35"  /f
 
 
-build:
-  verbosity: normal
+build_script:
+  - msbuild "c:\projects\NuGetTest\NuGetProvider\vs-csproj\NugetLightProvider.csproj" /verbosity:normal
 after_build:
```

Comment:
can you use rebase when merge your changes to development so that we can keep the history clean. You can do this in the github by click on Rebase.